src/document/docUtil.ts

```diff
@@ -0,0 +1,53 @@
+import { SelectionItem } from '../store';
+
+/**
+ * Finds the closest ancestor DOM element with the specified class.
+ */
+export function findClosestElWithClass(element: Element | null, className: string): Element | null {
+    for (let el = element; el && el !== ((document as unknown) as Element); el = el.parentNode as Element | null) {
+        if (el.classList && el.classList.contains(className)) {
+            return el;
+        }
+    }
+
+    return null;
+}
+
+/**
+ * Returns the page element and page number that the element is on.
+ * If not found return null/-1
+ */
+export function getPageInfo(element: Element | null): { pageEl: Element | null; page: number } {
+    const pageEl = findClosestElWithClass(element, 'page') || null;
```

Comment:
This function already returns `null` as its falsy value. Is the default value necessary?